validator/src/main/java/com/amazon/aoc/enums/GenericConstants.java

```diff
@@ -22,7 +22,7 @@
   // retry
   SLEEP_IN_MILLISECONDS("10000"), // ms
   SLEEP_IN_SECONDS("30"),
-  MAX_RETRIES("10"),
+  MAX_RETRIES("20"),
```

Comment:
nit: Can we rather increase the sleep time than mex_retries? Then we will see less logs in the terminal which is good in my opinion.

Reply:
good point, will change